CRITICAL - 2022-03-22 03:23:13 --> syntax error, unexpected variable "$save"
#0 C:\xampp\htdocs\ci_project\system\Autoloader\Autoloader.php(241): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs...')
#1 C:\xampp\htdocs\ci_project\system\Autoloader\Autoloader.php(219): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Models\\Demo...')
#2 C:\xampp\htdocs\ci_project\app\Controllers\Register.php(30): CodeIgniter\Autoloader\Autoloader->loadClass('App\\Models\\Demo...')
#3 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(832): App\Controllers\Register->save()
#4 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#5 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\ci_project\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
ERROR - 2022-03-22 04:35:34 --> Unknown column 'user_email' in 'where clause'
CRITICAL - 2022-03-22 04:35:34 --> Call to a member function getRow() on bool
#0 C:\xampp\htdocs\ci_project\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('nikhilkd@gmail....', 'user_email', Array, NULL)
#1 C:\xampp\htdocs\ci_project\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('email', 'email', 'nikhilkd@gmail....', Array, Array)
#2 C:\xampp\htdocs\ci_project\system\Controller.php(160): CodeIgniter\Validation\Validation->run()
#3 C:\xampp\htdocs\ci_project\app\Controllers\Register.php(29): CodeIgniter\Controller->validate(Array)
#4 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(832): App\Controllers\Register->save()
#5 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#6 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\ci_project\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
ERROR - 2022-03-22 04:36:35 --> Unknown column 'user_email' in 'where clause'
CRITICAL - 2022-03-22 04:36:35 --> Call to a member function getRow() on bool
#0 C:\xampp\htdocs\ci_project\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('nikhilkd@gmail....', 'user_email', Array, NULL)
#1 C:\xampp\htdocs\ci_project\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('email', 'email', 'nikhilkd@gmail....', Array, Array)
#2 C:\xampp\htdocs\ci_project\system\Controller.php(160): CodeIgniter\Validation\Validation->run()
#3 C:\xampp\htdocs\ci_project\app\Controllers\Register.php(29): CodeIgniter\Controller->validate(Array)
#4 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(832): App\Controllers\Register->save()
#5 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#6 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\ci_project\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-03-22 04:36:39 --> Array to string conversion
#0 C:\xampp\htdocs\ci_project\app\Controllers\Register.php(44): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Array to string...', 'C:\\xampp\\htdocs...', 44)
#1 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(832): App\Controllers\Register->save()
#2 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#3 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\ci_project\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2022-03-22 04:37:03 --> Array to string conversion
#0 C:\xampp\htdocs\ci_project\app\Controllers\Register.php(44): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Array to string...', 'C:\\xampp\\htdocs...', 44)
#1 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(832): App\Controllers\Register->save()
#2 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#3 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\ci_project\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2022-03-22 04:38:28 --> Undefined variable $data
#0 C:\xampp\htdocs\ci_project\app\Controllers\Register.php(42): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined varia...', 'C:\\xampp\\htdocs...', 42)
#1 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(832): App\Controllers\Register->save()
#2 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#3 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\ci_project\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2022-03-22 05:18:40 --> Invalid file: contact.php
#0 C:\xampp\htdocs\ci_project\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('contact.php')
#1 C:\xampp\htdocs\ci_project\system\Common.php(1089): CodeIgniter\View\View->render('contact', Array, true)
#2 C:\xampp\htdocs\ci_project\app\Controllers\Register.php(24): view('contact', Array)
#3 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(832): App\Controllers\Register->index()
#4 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#5 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\ci_project\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-03-22 05:20:16 --> Invalid file: id.php
#0 C:\xampp\htdocs\ci_project\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('id.php')
#1 C:\xampp\htdocs\ci_project\system\Common.php(1089): CodeIgniter\View\View->render('id', Array, true)
#2 C:\xampp\htdocs\ci_project\app\Controllers\Register.php(24): view('id', Array)
#3 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(832): App\Controllers\Register->index()
#4 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#5 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\ci_project\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-03-22 05:20:17 --> Invalid file: id.php
#0 C:\xampp\htdocs\ci_project\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('id.php')
#1 C:\xampp\htdocs\ci_project\system\Common.php(1089): CodeIgniter\View\View->render('id', Array, true)
#2 C:\xampp\htdocs\ci_project\app\Controllers\Register.php(24): view('id', Array)
#3 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(832): App\Controllers\Register->index()
#4 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#5 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\ci_project\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-03-22 05:20:34 --> Invalid file: .php
#0 C:\xampp\htdocs\ci_project\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('.php')
#1 C:\xampp\htdocs\ci_project\system\Common.php(1089): CodeIgniter\View\View->render('', Array, true)
#2 C:\xampp\htdocs\ci_project\app\Controllers\Register.php(24): view('', Array)
#3 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(832): App\Controllers\Register->index()
#4 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#5 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\ci_project\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-03-22 05:20:35 --> Invalid file: .php
#0 C:\xampp\htdocs\ci_project\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('.php')
#1 C:\xampp\htdocs\ci_project\system\Common.php(1089): CodeIgniter\View\View->render('', Array, true)
#2 C:\xampp\htdocs\ci_project\app\Controllers\Register.php(24): view('', Array)
#3 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(832): App\Controllers\Register->index()
#4 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#5 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\ci_project\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-03-22 05:21:16 --> Invalid file: DemoModel.php
#0 C:\xampp\htdocs\ci_project\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('DemoModel.php')
#1 C:\xampp\htdocs\ci_project\system\Common.php(1089): CodeIgniter\View\View->render('DemoModel', Array, true)
#2 C:\xampp\htdocs\ci_project\app\Controllers\Register.php(24): view('DemoModel', Array)
#3 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(832): App\Controllers\Register->index()
#4 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#5 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\ci_project\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-03-22 05:21:17 --> Invalid file: DemoModel.php
#0 C:\xampp\htdocs\ci_project\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('DemoModel.php')
#1 C:\xampp\htdocs\ci_project\system\Common.php(1089): CodeIgniter\View\View->render('DemoModel', Array, true)
#2 C:\xampp\htdocs\ci_project\app\Controllers\Register.php(24): view('DemoModel', Array)
#3 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(832): App\Controllers\Register->index()
#4 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#5 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\ci_project\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-03-22 05:24:21 --> Invalid file: DemoModel.php
#0 C:\xampp\htdocs\ci_project\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('DemoModel.php')
#1 C:\xampp\htdocs\ci_project\system\Common.php(1089): CodeIgniter\View\View->render('DemoModel', Array, true)
#2 C:\xampp\htdocs\ci_project\app\Controllers\Register.php(24): view('DemoModel', Array)
#3 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(832): App\Controllers\Register->index()
#4 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#5 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\ci_project\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-03-22 05:24:28 --> Invalid file: DemoModel.php
#0 C:\xampp\htdocs\ci_project\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('DemoModel.php')
#1 C:\xampp\htdocs\ci_project\system\Common.php(1089): CodeIgniter\View\View->render('DemoModel', Array, true)
#2 C:\xampp\htdocs\ci_project\app\Controllers\Register.php(24): view('DemoModel', Array)
#3 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(832): App\Controllers\Register->index()
#4 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#5 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\ci_project\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-03-22 06:25:11 --> syntax error, unexpected token "public", expecting end of file
#0 C:\xampp\htdocs\ci_project\system\Autoloader\Autoloader.php(241): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs...')
#1 C:\xampp\htdocs\ci_project\system\Autoloader\Autoloader.php(219): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(797): class_exists('\\App\\Controller...', true)
#4 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(406): CodeIgniter\CodeIgniter->startController()
#5 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\ci_project\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-03-22 06:25:35 --> syntax error, unexpected token "public", expecting end of file
#0 C:\xampp\htdocs\ci_project\system\Autoloader\Autoloader.php(241): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs...')
#1 C:\xampp\htdocs\ci_project\system\Autoloader\Autoloader.php(219): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(797): class_exists('\\App\\Controller...', true)
#4 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(406): CodeIgniter\CodeIgniter->startController()
#5 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\ci_project\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-03-22 06:25:42 --> syntax error, unexpected token "public", expecting end of file
#0 C:\xampp\htdocs\ci_project\system\Autoloader\Autoloader.php(241): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs...')
#1 C:\xampp\htdocs\ci_project\system\Autoloader\Autoloader.php(219): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(797): class_exists('\\App\\Controller...', true)
#4 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(406): CodeIgniter\CodeIgniter->startController()
#5 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\ci_project\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-03-22 06:28:06 --> syntax error, unexpected token "public", expecting end of file
#0 C:\xampp\htdocs\ci_project\system\Autoloader\Autoloader.php(241): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs...')
#1 C:\xampp\htdocs\ci_project\system\Autoloader\Autoloader.php(219): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(797): class_exists('\\App\\Controller...', true)
#4 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(406): CodeIgniter\CodeIgniter->startController()
#5 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\ci_project\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-03-22 06:28:07 --> syntax error, unexpected token "public", expecting end of file
#0 C:\xampp\htdocs\ci_project\system\Autoloader\Autoloader.php(241): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs...')
#1 C:\xampp\htdocs\ci_project\system\Autoloader\Autoloader.php(219): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(797): class_exists('\\App\\Controller...', true)
#4 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(406): CodeIgniter\CodeIgniter->startController()
#5 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\ci_project\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-03-22 06:29:07 --> Call to undefined function set_value()
#0 C:\xampp\htdocs\ci_project\system\View\View.php(210): include()
#1 C:\xampp\htdocs\ci_project\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#2 C:\xampp\htdocs\ci_project\system\Common.php(1089): CodeIgniter\View\View->render('register', Array, true)
#3 C:\xampp\htdocs\ci_project\app\Controllers\Register.php(11): view('register')
#4 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(832): App\Controllers\Register->index()
#5 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#6 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\ci_project\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-03-22 06:40:24 --> There is no data to insert.
#0 C:\xampp\htdocs\ci_project\system\BaseModel.php(718): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('insert')
#1 C:\xampp\htdocs\ci_project\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#2 C:\xampp\htdocs\ci_project\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#3 C:\xampp\htdocs\ci_project\app\Controllers\Register.php(39): CodeIgniter\BaseModel->save(Array)
#4 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(832): App\Controllers\Register->registerInsert()
#5 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#6 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\ci_project\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-03-22 06:46:30 --> Call to undefined method App\Controllers\Register::request()
#0 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(832): App\Controllers\Register->registerInsert()
#1 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#2 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\ci_project\index.php(37): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2022-03-22 06:48:18 --> Call to undefined method App\Controllers\Register::request()
#0 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(832): App\Controllers\Register->registerInsert()
#1 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#2 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\ci_project\index.php(37): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2022-03-22 06:50:07 --> Invalid file: success.php
#0 C:\xampp\htdocs\ci_project\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('success.php')
#1 C:\xampp\htdocs\ci_project\system\Common.php(1089): CodeIgniter\View\View->render('success', Array, true)
#2 C:\xampp\htdocs\ci_project\app\Controllers\Register.php(42): view('success')
#3 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(832): App\Controllers\Register->registerInsert()
#4 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#5 C:\xampp\htdocs\ci_project\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\ci_project\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
